plugins/templates-maven-plugin/src/main/java/com/google/cloud/teleport/plugin/maven/TemplatesStageMojo.java

```diff
@@ -258,6 +258,11 @@ protected String stageClassicTemplate(
     arguments.add(element("argument", "--templateLocation=" + templatePath));
     arguments.add(element("argument", "--project=" + projectId));
     arguments.add(element("argument", "--region=" + region));
+    if (imageSpec.getMetadata().isStreaming()) {
+      // Default to THROUGHPUT_BASED autoscaling for streaming classic templates
```

Comment:
so the user cannot change these settings?

Reply:
They can adjust maxNumWorkers, but not the autoscaling algorithm

Reply:
will `arguments.add(element("argument", "--maxNumWorkers=10"));` override the maxNumWorkers option specified by the user?

Reply:
Overriding maxNumWorkers still works. I updated to demonstrate this with a test below which produced this job - https://pantheon.corp.google.com/dataflow/jobs/us-central1/2024-04-04_09_33_22-17280287866962791451;graphView=0?project=cloud-teleport-testing&e=13802955&mods=logs_tg_staging&pageState=(%22dfTime%22:(%22l%22:%22dfJobMaxTime%22))